### PR TITLE
Add analytics service - persist feedback in Postgresql

### PR DESCRIPTION
**Pull request by pierDipi**
After executing a few of these commands (faking feedback from client):
```
curl -v -X POST -H "Content-Type: application/json" http://analytics-service-ai-demo.apps.<redacted>.origin-ci-int-gce.dev.rhcloud.com/feedbacks -d '{"feedback": 1, "upload_id": "uuu"}'
```

we can see data in postgresql:
```
ai-demo=> SELECT * FROM feedbacks;
 id | feedback | upload_id |         created_on         
----+----------+-----------+----------------------------
  1 |        1 | xyz       | 2023-09-28 16:23:03.888924
  2 |        1 | zzz       | 2023-09-28 16:23:13.389398
  3 |        1 | aaa       | 2023-09-28 16:27:17.055931
  4 |        1 | uuu       | 2023-09-28 16:27:21.582464
  5 |       10 | www       | 2023-09-28 16:30:01.901031

```

### CODE REVIEW

File: infra/openshift-manifests/ai-demo-namespace.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: ai-demo
\ No newline at end of file
+  name: ai-demo
+  labels:
+    bindings.knative.dev/include: "true"
```
Comment (aliok) [inline infra/openshift-manifests/ai-demo-namespace.yaml:6]: So, was this needed after all?

**Comment by aliok** [comment]
We have a "Feedback collection service" and an "analytics service" in the diagram. Which one is this service?

Let's talk and update the diagram if you have something else in your mind. Or, maybe I am reading things wrong.

**Comment by aliok** [comment]
I am merging this. You can just continue from here.
I don't want to run into conflicts.

File: infra/openshift-manifests/ai-demo-namespace.yaml (hunk 1 above)
Comment (pierDipi) [reply infra/openshift-manifests/ai-demo-namespace.yaml:6]: Yes, not sure why it sometimes works

**Pull request merged**